script/Arcueid/Day 5/05_00_ARC05_5B.txt

```diff
@@ -67,17 +67,17 @@ So that's what she meant by 'dropping them underground'.
 [sha:d82b663aed982652af1ad88715c388292c5e45a0]{
 -- Page 427, Offset 9715.
 -- 深さ、実に40メートル以上の地盤沈下。
-The land sunk over forty meters.
+The land has sunk over forty meters.
 }
 [sha:1cdc98b538cace1716e3874c0e76f06411ef0704]{
 -- Page 428, Offset 9716.
 -- 壮大な自然の営みを見た気分だ。アマゾンの密林とか、ナイアガラの滝とか、そのレベルの雄大さ。
-It felt like watching an incredible force of nature. Its grandeur is on the same level as the Amazon jungle or Niagara Falls.
+It felt like watching an incredible force of nature. Its grandeur is on the same level as the Amazon jungle or the Niagara Falls.
```

Comment:
no `the` before Niagara

Reply:
yeah, i had a momentary lapse of reason on this one but it's definitely "the"-less